src/client/state/actions/allActions.js

```diff
@@ -1,7 +1,23 @@
 import { addName, addUserName, setNumber } from '../reducers/namesReducer';
+import { addItem, subtractItem, UpdateItemPrice, UpdateItemDescription, resetItemQuantity } from '../reducers/menuItemReducer';
+import { UpdateItemName, UpdateItemId, UpdateItemQuantity, UpdateTotalPrice, UpdateRestaurantId } from '../reducers/orderItemReducer';
+import { addItemToOrders, addToPriceTotal } from '../reducers/allOrderItemsReducer';
 
 const allActions = {
-  addName, addEmail
+  addName,
+  // addEmail,
```

Comment:
Ran into an issue where addEmail wasnt imported so commented it out for now until login page updates it?

Reply:
Yeah, my bad, I added that action and forgot to edit it out before I merged earlier.